### PR TITLE
Correct spelling mistakes

### PR DESCRIPTION
**Pull request by yuxiaobo96**
Signed-off-by: yuxiaobo <yuxiaobogo@163.com>

### CODE REVIEW

File: upup/pkg/fi/context.go [hunk 1]
```diff
@@ -158,7 +158,7 @@ func (c *Context) Render(a, e, changes Task) error {
 				b.WriteTo(out)
 
 				if *lifecycle == LifecycleExistsAndValidates {
-					return fmt.Errorf("Lifecycle set to ExistsAndValidates, but object did not match")
+					return fmt.Errorf("lifecycle set to ExistsAndValidates, but object did not match")
```
Comment (bittopaz) [inline upup/pkg/fi/context.go]: Can you tell the reason of change the word to lowercase, please?

Comment (yuxiaobo96) [reply upup/pkg/fi/context.go]: This is a static inspection specification.

Comment (yuxiaobo96) [reply upup/pkg/fi/context.go]: As you can see, it is just a specification.

**Pull request merged**